src/styles/_section.scss

```
@@ -0,0 +1,55 @@
+.section {
+  width: 100%;
+  overflow: hidden;
+  &__wrapper {
+    margin: 0 auto;
+    max-width: $wrapper-width;
```

add empty line before element
```suggestion
.section {
  width: 100%;
  overflow: hidden;
  
  &__wrapper {
    margin: 0 auto;
    max-width: $wrapper-width;
```

---

виправив